pkg/reconciler/knativeeventing/common/extensions_test.go

```diff
@@ -61,20 +61,20 @@ func TestTransformers(t *testing.T) {
 
 	results, err := platform.Transformers(kubeclient.Get(ctx), ke, logger)
 	assertEqual(t, err, nil)
-	// By default, there are 4 functions.
-	assertEqual(t, len(results), 4)
+	// By default, there are 5 functions.
+	assertEqual(t, len(results), 5)
```

Comment:
This seems a bit fragile, is there any way to test that the functions are doing what we'd expect?

Reply:
To me, this only tests the number of functions it calls.
For tests verifying each function, we have test cases for each function.